src/main/windows/callsWidgetWindow.ts

```diff
@@ -378,15 +378,17 @@ export class CallsWidgetWindow {
     private handleGetDesktopSources = async (event: IpcMainInvokeEvent, opts: Electron.SourcesOptions) => {
         log.debug('handleGetDesktopSources', opts);
 
-        if (event.sender.id !== this.mainView?.webContentsId) {
+        // For Calls we make an extra check to ensure the event is coming from the expected window (main view).
+        // Otherwise we want to allow for other plugins to ask for screen sharing sources.
+        if (this.mainView && event.sender.id !== this.mainView.webContentsId) {
             log.warn('handleGetDesktopSources', 'Blocked on wrong webContentsId');
```

Comment:
If we're throwing errors here, these logs should be unnecessary as the `CriticalErrorHandler` should catch and log these.